scrapyd_k8s/logging_config.py

```diff
@@ -1,15 +1,15 @@
 import logging
 import sys
 
-def setup_logging(logging_level):
-    if not logging_level:
-        logging_level = 'INFO'  # Default to INFO if logging_level is None
+def setup_logging(log_level):
+    if not log_level:
+        log_level = 'INFO'  # Default to INFO if logging_level is None
```

Comment:
We already have a default, right? So we can omit it here.

Reply:
Right, double check is not needed here, removed it.